### PR TITLE
FIX Documentation to adapt to FIWARE TOC

### PR DESCRIPTION
**Pull request by dmoranj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -475,116 +91,6 @@ To generate TAP report in `report/test/unit_tests.tap`, type
...
-```bash
-grunt site
-```
-
-This command will only work after the developer has executed init-dev-env (that's the goal that will create the detached site).
-
-This command will also launch the coverage, doc and complexity task (see in the above sections).
+##  <a name="development"/> Development documentation
 
+Information about developing for the JSON IoTAgent can be found at the corresponding section of the [User & Programmers Guide](docs/usermanual.md).
```
Comment (jmcanterafonseca) [inline README.md]: manual

Comment (dmoranj) [inline README.md]: Fixed in 			67c5e18

File: README.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+* [API Overview] (#apioverview)
+* [API Reference Documentation] (#apireference)
 * [Command Line Client](#client)
-* [Development documentation](#development)
+* [Testing] (#testing)
+* [Development] (#development)
 
-## <a name="overview"/> Overview
+## <a name="description"/> Description
 This IoT Agent is designed to be a bridge between an MQTT+JSON based protocol and the OMA NGSI standard used in FIWARE.
```
Comment (dmoranj) [inline README.md]: Fixed in 			67c5e18

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**